infra/cluster-init/init.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+set -e
+
+function parse_args {
+    local args=(
+        "ISTIO_BASE_HELM_VALUES_PATH"
+        "ISTIO_DISCOVERY_HELM_VALUES_PATH"
+        "ISTIO_INGRESS_HELM_VALUES_PATH"
+    )
+    for arg in ${args[@]}; do
+        [[ -z "${!arg}" ]] && echo "${arg} environment variable is not set." && exit 1
+    done
+
+    echo "Args check done."
+}
+
+function print_usage {
+    echo "Usage: $0 <kubeconfig file>"
+    echo "  Environment Variables:"
+    echo "    ISTIO_BASE_HELM_VALUES_PATH       Helm values of Istio Base."
+    echo "    ISTIO_DISCOVERY_HELM_VALUES_PATH  Helm values of Istio Discovery (istiod)."
+    echo "    ISTIO_INGRESS_HELM_VALUES_PATH    Helm values of Istio Ingress Gateway."
+    echo
+    echo "  Optional Environment Variables:"
+    echo "    ISTIO_VERSION                     Istio version, default: 1.9.9."
+    echo "    KNATIVE_VERSION                   Knative version, default: 0.18.3."
+    echo "    KNATIVE_ISTIO_VERSION             Knative Istio version, default: 0.18.1."
+}
+
+function install_istio {
+    echo "Installing Istio."
+
+    local istio_version=${ISTIO_VERSION:-1.9.9}
+    curl -L https://istio.io/downloadIstio | ISTIO_VERSION=${istio_version} TARGET_ARCH=x86_64 sh -
+    kubectl create namespace istio-system --dry-run=client -o yaml | kubectl apply -f -
+    helm install istio-base "istio-${istio_version}/manifests/charts/base" \
```

Comment:
I'm caving in and have swapped it to istioctl. Reason being you can't really set global values across all the charts easily because there is no "uber" istio chart.